src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -93,9 +98,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    String text = Jsoup.clean(request.getParameter("text"), Whitelist.none().addTags("strong", "i", "ins"));
```

Comment:
I think you actually don't want this anymore.  Now that your HTML handling is happening at display-time, you can leave the original code that just removed all HTML markup.  I guess that's another benefit of this way of setting things up that I hadn't even thought of :-)